CHANGELOG.md

```diff
@@ -3,6 +3,39 @@
 Nothing Yet!
 
 
+# Version 0.16.0 (2024-06-14)
+
+This release introduces some new [supplychain security features](https://opensource.axo.dev/cargo-dist/book/supplychain-security/index.html), and fixes some bugs.
+
+## GitHub Artifact Attestations
+
+With the new [`github-attestations = true` setting](https://opensource.axo.dev/cargo-dist/book/supplychain-security/attestations/github.html) you can opt into GitHub's experimental artifact attestation system. In the future this may become enabled by default.
+
+* @dunxen [feat: add support for github artifact attestations](https://github.com/axodotdev/cargo-dist/pull/1012)
+* @ashleygwilliams [doc: cleanup supplychain security docs](https://github.com/axodotdev/cargo-dist/pull/1123)
+
+
+## Reducing Excessive Third-Party Actions
```

Comment:
we might not need the word excessive here- i think you strike the right tone with the body, but def want to avoid the action author feeling attacked